source/hardware/controllers/native_instruments_traktor_kontrol_s3.rst

```diff
@@ -0,0 +1,165 @@
+
+.. versionadded:: 2.3.0
+
+The Kontrol S3 is an introductory 4 deck controller with good build
+quality and integrated sound card. This is the first controller released
+with the "S3" name.
+
+The Kontrol S3 can run from USB bus power. Using the separate power
+supply increases the brightness of the LEDs, which is helpful for using
+it in daylight, and increases the volume of the headphone output.
+
+  - `Manufacturer's product
+    page <https://www.native-instruments.com/en/products/traktor/dj-controllers/traktor-kontrol-s3/>`__
+
+Compatibility
+-------------
+
+Controller
+~~~~~~~~~~
+
+The Kontrol S3 is a USB class compliant audio and :term:`HID` device,
+so it is compatible with Mixxx without any proprietary drivers on
+GNU/Linux and Mac OS X. On Windows, it is recommended to install the
+`driver from Native
+Instruments <https://www.native-instruments.com/en/support/downloads/drivers-other-files/#traktorkontrols3>`__
+and select the ASIO sound API in the Sound Hardware section of Mixxx's
+Preferences.
+
+With the S3 plugged in, the device is listed as an available
+controller in Mixxx's Preferences. The controller uses
+HID for the knobs, buttons, and other components on the device, so the
+mapping can only be loaded when you select the HID device on the left
+side of Mixxx's Preferences.
+
+Mapping description
+-------------------
+
+Note that Mixxx doesn't have the concept of a single "master" deck for
+sync. Instead, push and hold the :hwlabel:`SYNC` button to "lock" sync on for all
+decks you want to remain in sync. Or you can push :hwlabel:`SHIFT` + :hwlabel:`SYNC` to lock
+sync on. Refer to `the Mixxx
+manual <../../chapters/djing_with_mixxx.html#master-sync>`__
+for details.
+
+.. hint:: There are two :hwlabel:`SHIFT` buttons, one for each side of the
+          controller. Make sure you press the :hwlabel:`SHIFT` button on the
+          same side as the control you want to modify.
+
+Mixer
+~~~~~
+
+  - Gain, equalizer high/mid/low, and cue (headphones)
+    behave as labelled.
+  - :hwlabel:`FX Enable` buttons: See Effect section below.
+  - Filter: controls QuickEffect superknob. This controls the Filter
+    effect by default, but a different effect can be chosen in the
+    Equalizer section of Mixxs's Preferences.
+  - :hwlabel:`EXT`: The :hwlabel:`EXT` button changes the fourth channel pregain (knob), pfl, and volume (slider) adjustments to operate with the Microphone input.  The microphone does not respond to EQ or effects.
+  - :hwlabel:`SHIFT` + :hwlabel:`EXT`: Switches input from Mic to Line and back again.
+
+The Master Volume knob on the S3 controls the volume of the S3's master
+output in hardware, so it does not affect the software master gain knob
+in Mixxx by default. You can hold :hwlabel:`SHIFT` and turn the knob to adjust Mixxx's
+master gain.  Note that this will still also adjust the hardware gain, so
+after you release :hwlabel:`SHIFT` you'll need to adjust the knob again.
```

Comment:
```suggestion
The Main Volume knob on the S3 controls the volume of the S3's main
output in hardware, so it does not affect the software main output gain knob
in Mixxx by default. You can hold :hwlabel:`SHIFT` and turn the knob to adjust Mixxx'
main output gain.  Note that this will still also adjust the hardware gain, so
after you release :hwlabel:`SHIFT` you'll need to adjust the knob again.
```

Reply:
thanks for these!